smart-contracts/test/test_security.js

```diff
@@ -70,10 +73,31 @@ describe("Security Test", function () {
         owner,
         userOne,
         userThree,
-        pauser.address
+        pauser.address,
+        networkDescriptor
       );
     });
 
+    it("should allow operator to call reinitalize after initialization", async function () {
+      await expect(state.bridgeBank.connect(operator).reinitialize(
```

Comment:
We should add another test that checks if you try to call reinitialize a second time it fails and reverts as you should only be able to call this function a single time.